### PR TITLE
#bbcode_formatted? method for String

### PR DESCRIPTION
**Pull request by iarie**
Hello, in my project i need to check if string has bb-tags so i can choose decorator method for it, so i propose to add a simple method to your gem: it checks if any bb-tags present in given string.
And i also found some mistypes in ```@@tags``` examples

### CODE REVIEW

File: lib/bb-ruby.rb [hunk 1]
```diff
@@ -315,6 +315,11 @@ def tag_list
       @@tags
     end
 
+    # Checks if text contains any bb tag
+    def any_bb_tag?(text)
```
Comment (cpjolicoeur) [inline lib/bb-ruby.rb]: Can we please change this method name to the plural form? `#any_bb_tags?`.

File: lib/bb-ruby.rb [hunk 2]
```diff
@@ -118,7 +118,7 @@ module BBRuby
       /\[dd\](.*?)\[\/dd\]/mi,
       '<dd>\1</dd>',
       'Definition definitions',
-      '[dd]my definition[/dd',
+      '[dd]my definition[/dd]',
```
Comment (cpjolicoeur) [inline lib/bb-ruby.rb]: Thanks for the above changes.  Can you make these corrections a separate commit from the new feature added below?  One commit for the "fixes" and one for the new feature?  Thanks

**Comment by iarie** [comment]
ok, i did separate commits and change method name

**Comment by cpjolicoeur** [comment]
@iarie v1.3.0 released with these changes.  Thanks
